test/scoring_test.dart

```diff
@@ -59,13 +59,13 @@ void main() {
         expect(() => summary.simpleScore(49), throwsRangeError);
         expect(() => summary.simpleScore(151), throwsRangeError);
 
-        expect(summary.simpleScore(150), 1);
+        expect(summary.simpleScore(150) as double, 1);
```

Comment:
why is the `as double` needed here?

Reply:
This ensures that the runtime type of `simpleScore` is a double – and throws otherwise.

It acts like a regression test for the changed behavior

Reply:
I think `expect(summary.simpleScore(150), 1.0);` should have the same result. I'm fine either way.

Reply:
It would – because the comparison between of `1.0` or `1` with `1` is the same.

This is just checking that everything is a `double`